src/main/java/org/takes/rq/RqMethod.java

```diff
@@ -94,6 +96,13 @@
     final class Base extends RqWrap implements RqMethod {
 
         /**
+         * HTTP method line pattern.
+         */
+        private static final Pattern PATTERN = Pattern.compile(
+            "(GET|POST|PUT|DELETE|HEAD|OPTIONS|PATCH) [^ ]+( [^ ]+){0,1}"
```

Comment:
@dmzaytsev I allow only one space in the regex part after the URI group of characters: ( [^ ]+)
All of the other characters must be non-space ones: [^ ]